src/Avalonia.Controls/Shapes/Polyline.cs

```diff
@@ -14,10 +14,15 @@ static Polyline()
             AffectsGeometry<Polyline>(PointsProperty);
         }
 
+        public Polyline()
+        {
+            Points = new Points();
+        }
```

Comment:
Not related to this PR, but changes nullability of points property to be consistent with PolyLineSegment/Geometry.